### PR TITLE
Bump googleapis to latest version

### PR DESCRIPTION
**Pull request by grantfayvor**
- bump googleapis to latest version to resolve vulnerability issues
- fix backward compatibilities issues

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -133,7 +133,7 @@ class GoogleAuth extends Base {
       const oAuth2Client = await this._getOAuth2Client()
       oAuth2Client.setCredentials(token)
       const oauth2 = google.oauth2({ version: 'v2', auth: oAuth2Client })
-      oauth2.userinfo.v2.me.get(
+      oauth2.userinfo.get(
         (err, data) => {
           if (err) reject(new Error('AUTH_FAC_ERROR_ASK_EMAIL:' + err.toString()))
           else resolve(data.data.email)
```
Comment (ezewer) [inline index.js:139]: Can we add here:
resolve(data && data.data && data.data.email)
Just in case

**Review by ezewer — COMMENTED** [review]
Code looks good!
Please bump version to 1.0.3 and added a small request

File: index.js (hunk 1 above)
Comment (grantfayvor) [reply index.js]: done.

**Review by ezewer — APPROVED** [review]
Looks good to me!

**Pull request merged**